cmake/Modules/Findswig.cmake

```diff
@@ -0,0 +1,44 @@
+add_executable(swig IMPORTED)
+
+find_program(SWIG_EXECUTABLE NAMES swig3.0 swig2.0 swig)
+
+find_package_handle_standard_args(SWIG DEFAULT_MSG
+  SWIG_EXECUTABLE
+  )
+
+if(NOT SWIG_EXECUTABLE)
+  message(STATUS "Installed package SWIG not found. It will be pulled to " ${SWIG_DIR} " at compile time.")
```

Comment:
`SWIG_DIR` should be set before this line. Probably this message has to be displayed on line 34.